docs/templates.md

```diff
@@ -256,8 +255,8 @@ practices:
 - Ensure the resource has `curl` installed
 - Ensure the resource can reach your Coder URL
 - Manually connect to the resource (e.g., `docker exec` or AWS console)
-  - The Coder agent logs are typically stored in (`/var/log/coder-agent.log`)
-
+  - The Coder agent logs are typically stored in `/var/log/coder-agent.log`
+  - The Coder agent startup script logs are typically stored in `/var/log/coder-startup-script.log`
```

Comment:
We should just say stored, not typically

Reply:
What about Windows @kylecarbs?